internal/component/database_observability/mysql/collector/query_tables.go

```diff
@@ -117,7 +131,7 @@ func (c *QueryTables) Stop() {
 	c.cancel()
 }
 
-func (c *QueryTables) fetchQueryTables(ctx context.Context) error {
+func (c *QueryTables) tablesFromQuerySamples(ctx context.Context) error {
```

Comment:
What is the rationale for this new name? It doesn't seem related to query samples as we have modelled it in the component.

Reply:
Well these are actualy samples though, as they're coming from `events_statements_summary_by_digest.query_sample_text`.

Anyway I wanted to make this clearly distinct from the other methods that deals with prepared statements rather than regular queries.

Happy to use a name recommendation.